doc/concepts/embeddings.rst

```diff
@@ -0,0 +1,57 @@
+
+
+Basis Embedding
+^^^^^^^^^^^^^^^^^^^^
+
+Basis embedding associates each input with a computational basis state of a qubit system. Therefore, classical data has to be in the form of binary strings. The embedded quantum state is the bit-wise translation of a binary string to the corresponding states of the quantum subsystems. For example, :math:`x=1001` is represented by the 4-qubit quantum state :math:`\mid 1001 \rangle`. Hence, one bit of classical information is represented by one quantum subsystem.
+
+Let's consider the classical dataset :math:`\mathcal{D}` mentioned above. For basis embedding, each example has to be a N-bit binary string; :math:`x^{(m)}=(b_1,\ldots,b_N)` with :math:`b_i \in \{0,1\}` for :math:`i=1,\ldots,N`. Assuming all features are repesented with unit binary precision (one bit), each input example :math:`x^{(m)}` can be directly mapped to the quantum state :math:`\mid x^{(m)}\rangle`. **This means that the number of quantum subsystems,** :math:`\bm{n}` **, must be at least equal to** :math:`\bm{N}`. An entire dataset can be represented in superpositions of computational basis states as
+
+
+.. math:: \mid \mathcal{D} \rangle = \frac{1}{\sqrt{M}} \sum_{m=1}^{M} \mid x^{(m)} \rangle.
+
+For example, let's say we have a classical data set containing two examples :math:`x^{(1)}=01` and :math:`x^{(2)}=11`. The corresponding basis encoding uses two qubits to represent :math:`\mid x^{(1)} \rangle=\mid 01 \rangle` and :math:`\mid x^{(2)} \rangle=\mid 11 \rangle` 
+
+.. math:: \mid \mathcal{D} \rangle = \frac{1}{\sqrt{2}}\mid 01 \rangle + \frac{1}{\sqrt{2}} \mid 11 \rangle.
+
+.. note:: For :math:`N` bits, there are :math:`2^N` possible basis states. Given :math:`M \ll 2^N`, the basis embedding of :math:`\mathcal{D}` will be sparse :cite:`Mariabook2018`. 
+
+
+Amplitude Embedding
+^^^^^^^^^^^^^^^^^^^^
+
+In amplitude embedding technique, data is encoded into the amplitudes of a quantum state. This can be easily understood if we concatenate all the input examples :math:`x^{(m)}` together into one vector, i.e., 
+
+.. math:: \alpha = C_n \{ x^{(1)}_1, \ldots, x^{(1)}_N, x^{(2)}_1, \ldots, x^{(2)}_N, \ldots, x^{(M)}_1, \ldots, x^{(M)}_N \},
+ 
+where :math:`C_n` is the normalization constant; this vector must be normalized :math:`\mid\alpha\mid^2=1`. The input data can now be represented in the computational basis as
+
+.. math:: \mid \mathcal{D} \rangle = \sum_{i=1}^{2^n} \alpha_i \mid i \rangle,
+
+where :math:`\alpha_i` are the elements of amplitude vector :math:`\alpha` and :math:`\mid i \rangle` are the computational basis states corresponding to the number of quantum subsystems that are used. The number of amplitudes to be encoded are :math:`N \times M`. As the number of quantum subsystems, :math:`n`, provide :math:`2^n` amplitudes, **amplitude embedding requires** :math:`\bm{n \geq \log_2({NM})}`  **quantum subsystems.**  
```

Comment:
This argument only seems to apply to qubits, so we might as well call them that directly
```suggestion
where :math:`\alpha_i` are the elements of amplitude vector :math:`\alpha` and :math:`\mid i \rangle` are the computational basis states corresponding to the number of quantum subsystems that are used. The number of amplitudes to be encoded are :math:`N \times M`. A system of :math:`n` qubits provides :math:`2^n` amplitudes, so **amplitude embedding requires** :math:`\bm{n \geq \log_2({NM})}`  **qubits.**  
```